components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -2806,6 +2832,10 @@ public String addIdPWithResourceId(IdentityProvider identityProvider, int tenant
                     + tenantId, e);
         } catch (ConnectorException e) {
             throw new IdentityProviderManagementException("An error occurred while filtering IDP properties.", e);
+        } catch (SecretManagementException e) {
+            throw new IdentityProviderManagementException("An error occurred while storing encrypted IDP secrets of " +
+                    "Identity provider : " +identityProvider.getIdentityProviderName() + " in tenant : "
```

Comment:
```suggestion
                    "Identity provider : " + identityProvider.getIdentityProviderName() + " in tenant : "
```

Reply:
Resolved in [dcc8602](https://github.com/wso2/carbon-identity-framework/pull/4430/commits/dcc86027c74a591fab059fd74e1cf0b14751330a)